src/js/library-page-only/filter.js

```diff
@@ -10,7 +10,7 @@ const DATA_KEY1 = 'watched';
 const DATA_KEY2 = 'queue';
 
 export function renderWatched() {
-  cardsRenderDivRef.innerHTML = '';
+  cardsRenderDivRef.innerHTML = `Its empty here, add movies you've already watched.`;
```

Comment:
Text appear when it is empty, that's okay. But this text have to shown ass well when user delete all cards from watched,